CSS/Redes Sociales/css/style.css

```diff
@@ -0,0 +1,165 @@
+/*!
+ * All Rights Reserved
+ * This software is proprietary information of
+ * Intelligent Sense
+ * Use is subject to license terms.
+ * Filename: style.scss
+ */
+/*
+ * Author:      cviquez@intelligentsense.com
+ * Date:        03/04/2019
+ * Description: Main COMPASS file. Import all the partials you need here.
+ */
+/**
+* Mixin for image replacement
+*/
+/* ==========================
+    Column main definition
+=============================*/
+
+.main-container {
+  margin: 25px;
+  border: 25px;
+  padding: 25px;
+  content: 25px;
+}
+
+
+/* ==========================================================================
+   Independent Columns fonts
+   ========================================================================== */
+.facebook  {
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.twitter  {
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+
+.instagram  {
+    font-family: 'Nunito', sans-serif;  
+}
+
+.fab  {
+    font-size: 48px;
+    position: absolute;
+    margin-left: 100px;
+    bottom: 20px;
+}
+
+/* ==========================================================================
+   Icon Style
+   ========================================================================== */
+
+a  {
+    color: rgb(253, 253, 253);
+    text-decoration: none;
+}
+
+a:hover  {
+    color: burlywood;
+
+}
+
+/* ==========================================================================
+   Media Queries
+   ========================================================================== */
+
+
+/* =====================
+   Mobile phones
+  ====================*/ 
```

Comment:
![image](https://user-images.githubusercontent.com/6954103/55746000-de1fdb00-59f5-11e9-9f9d-7a341054ad25.png)

I think it will be necessary to check the text area. If you resize the font or make it have a scroll on overflow, it will get better. This because on mobile view the text gets under the icon.